nucypher/config/characters.py

```diff
@@ -157,9 +157,10 @@ def __init__(self,
         super().__init__(*args, **kwargs)
         self.m = m or self.DEFAULT_M
         self.n = n or self.DEFAULT_N
-        if not self.federated_only:
-            self.rate = rate
-            self.duration_periods = duration_periods
+
+        # if not self.federated_only:  # TODO: why not?
```

Comment:
Probably deserves an issue, or a resolution of some sort.

Reply:
With the tests passing I'm inclined to just remove the TODO. objections?